u2f-gae-demo/war/js/u2fdemo.js

```diff
@@ -75,10 +75,15 @@ function hideMessage() {
 
 function highlightTokenCardOnPage(token) {
   console.log(token);
-  var cardContent = $("#" + token.public_key).find(".cardContent");
-  
-  cardContent.addClass("highlight");
-  window.setTimeout(function() { cardContent.removeClass("highlight", 2000); }, 500 );
+
+  var cardChildren = document.getElementById(token.public_key).children;
+  for (i = 0; i < cardChildren.length; i++) {
+    if (cardChildren[i].className == "cardContent") {
+      cardChildren[i].className += " highlight";
+      window.setTimeout(function() { cardChildren[i].className = "cardContent"; }, 500);
```

Comment:
This seems a bit more brittle than the previous code.  That is, the previous code found elements which had the "cardContent" class name (but could also have other class names as well).  It then appended the class name and then removed it.  This code seems to make an explicit comparison in the search "==".  Why not do a substring match so that if there were multiple classes than we would still find the class.  Similarly for the removal of the class -- I would save the class name as it was before you appended the highlight class name and then just restore it afterwards.  What do you think?